contribs/gnofaucet/serve.go

```diff
@@ -187,14 +220,18 @@ func execServe(ctx context.Context, cfg *serveCfg, io commands.IO) error {
 		),
 	)
 
-	// Start throttled faucet.
+	// Create cooldown limiter
 	st := newIPThrottler(defaultRateLimitInterval, defaultCleanTimeout)
+	// Start throttled faucet.
+	cooldownLimiter := NewCooldownLimiter(defaultRateLimitInterval, cfg.dbPath)
```

Comment:
Do we want to reuse the `defaultRateLimitInterval` value from the IP throttler for the cooldown limiter? 
Do these two mechanisms have exactly the same function? Or would it be relevant to add a dedicated flag for the `cooldownDuration`?